christfellowshipchurch/web-app-v2
# What to Expect Section

### About
This PR implements the new What to Expect at This Location Section under the set a reminder section so new people can see what to expect when visiting a location. The are videos for most locations, this section will not appear in the locations that do not have a video.

### Test Instructions
1. Go to `/locations` on the testing link and select a location that has a video, for example Palm Beach Gardens, and scroll down to after Set a Reminder, there should be a What to Expect at Palm Beach Gardens Section.
2. Go to `/locations` on the live website, select the same location and scroll down to after Set a Reminder, you should not see that section here.
3. Make sure that the section does not show in the locations that do not have videos.

### Screenshots
<img width="1439" alt="Screenshot 2023-09-13 at 1 56 11 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/72882da9-e516-4ec8-8e4b-0edef8e96ab7">


### Closes Tickets
[CFDP-2731](https://christfellowshipchurch.atlassian.net/browse/CFDP-2731)


[CFDP-2731]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2731?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
